packages/sass/README.md

```diff
@@ -7,32 +7,24 @@ This is the `sass` block providing Sass support for webpack. Uses `node-sass` vi
 
 ## Usage
 
-<!-- prettier-ignore-start -->
-```js
-const { createConfig } = require('@webpack-blocks/webpack')
-const sass = require('@webpack-blocks/sass')
-
-module.exports = createConfig([
-  sass(/* node-sass options */)
-])
-```
-<!-- prettier-ignore-end -->
-
-Use `match()` to explicitly specify on which files to apply the block:
-
 <!-- prettier-ignore-start -->
 ```js
 const { createConfig, match } = require('@webpack-blocks/webpack')
+const { css } = require('@webpack-blocks/assets')
 const sass = require('@webpack-blocks/sass')
 
 module.exports = createConfig([
   match(['*.scss', '!*node_modules*'], [
+    css(),
     sass(/* node-sass options */)
   ])
 ])
 ```
 <!-- prettier-ignore-end -->
 
+**NOTE**: the `css()` block by default works on `.css` files, so you have to make it accept `.scss`
```

Comment:
I‘d like to change the wording here:
"Note: Use `match()` here to apply the `css()` block to `.scss` files"?

Reply:
Sure!